nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/pom.xml

```diff
@@ -257,19 +257,15 @@
                     <linkPath>${libnd4jhome}/blasbuild/cpu/blas</linkPath>
                     <buildResources>
                         <buildResource>/${javacpp.platform.library.path}/</buildResource>
-                        <buildResource>/org/bytedeco/mkldnn/${javacpp.platform}/</buildResource>
```

Comment:
This should be replaced by the full version of MKL instead. There's no reason to not use it by default.